Microsoft.Toolkit.Win32.UI.Controls/WPF/WebView/WebView.cs

```diff
@@ -0,0 +1,1046 @@
+        /// the length of the scheme specified in <paramref name="source" /> exceeds 1023 characters, or
+        /// there is an invalid character sequence in <paramref name="source" />, or
+        /// the MS-DOS path specified in <paramref name="source" /> must start with c:\\.</exception>
+        public void Navigate(string source)
+        {
+            VerifyAccess();
+            Verify.IsNotNull(_webViewControl);
+            _webViewControl?.Navigate(source);
+        }
+
+        /// <summary>
+        /// Loads the HTML content at the specified Uniform Resource Identifier (URI).
+        /// </summary>
+        /// <param name="source">The Uniform Resource Identifier (URI) to load.</param>
+        /// <seealso cref="Uri" />
+        /// <see cref="Navigate(Uri)" /> is asynchronous. Use the <see cref="NavigationCompleted" /> event to detect when
+        /// navigation has completed.
+        public void Navigate(Uri source)
+        {
+            VerifyAccess();
+            Verify.IsNotNull(_webViewControl);
+
+            // TODO: Support for pack://
+            _webViewControl?.Navigate(source);
+        }
+
+        /// <summary>
+        /// Loads the specified HTML content as a new document.
+        /// </summary>
+        /// <param name="text">The HTML content to display in the control.</param>
+        public void NavigateToString(string text)
+        {
+            VerifyAccess();
+            Verify.IsNotNull(_webViewControl);
+            _webViewControl?.NavigateToString(text);
+        }
+
+        /// <summary>
+        /// Reloads the current <see cref="Source" /> in the <see cref="WebView"/>.
+        /// </summary>
+        /// <remarks>If the current source was loaded via <see cref="Navigate(Uri)" />, this method reloads the file without forced cache validation by sending a <c>Pragma:no-cache</c> header to the server.</remarks>
+        public void Refresh()
+        {
+            VerifyAccess();
+            Verify.IsNotNull(_webViewControl);
+            _webViewControl?.Refresh();
+        }
+
+        /// <summary>
+        /// Halts the current <see cref="WebView"/> navigation or download.
+        /// </summary>
+        public void Stop()
+        {
+            VerifyAccess();
+            Verify.IsNotNull(_webViewControl);
+            _webViewControl?.Stop();
+        }
+
+        protected override void Initialize()
+        {
+            Verify.IsNull(_process);
+
+            OSVersionHelper.ThrowIfBeforeWindows10RS4();
+
+            _process = new WebViewControlProcess();
+            Verify.IsNotNull(_process);
+
+            // TODO: Attach WebView to existing process
+            Dispatcher.InvokeAsync(async () =>
+            {
+                Verify.IsNotNull(_process);
+
+                var handle = ChildWindow.Handle;
+                var bounds = new Windows.Foundation.Rect(0, 0, RenderSize.Width, RenderSize.Height);
```

Comment:
@nmetulev Not sure how to get the position for the control since layout has not yet occurred. I've hooked over places where we would get this, but we make a bounds update at least twice in this scenario.